### ISSUE TITLE
Image ChoiceGroup text clips on overflow

### ISSUE THREAD

**Issue by Jahnp** [OfficeDev/office-ui-fabric-react#11860]
### Environment Information

- **Package version(s)**: 7.88.0 (on the main uifabric.io website)
- **Browser and OS versions**: Windows 10 Chromium Edge

### Please provide a reproduction of the bug in a codepen:
https://codepen.io/jahnp/pen/RwPbbMb?editable=true

This currently repros on the website@OUFR v7.88.0, on the ChoiceGroup page:
![image](https://user-images.githubusercontent.com/307118/73700930-d6c07100-469c-11ea-8762-95c7b6a57ad5.png)

#### Actual behavior:
Overflow text is cut off and parts of letters are not visible, even if it would fit within the choice field container. This is likely to prevent overflow text from "touching" the edge of the container, which might be unattractive for longer texts, but it looks odd when the strings are shorter.

#### Expected behavior:
Overflow text for shorter strings are handled more gracefully, or a title/tooltip is provided to handle overflow for longer strings.

### Priorities and help requested:

Are you willing to submit a PR to fix? Yes, but I don't have a lot of personal bandwidth (opening on behalf of a colleague)

Requested priority: Normal

Products/sites affected: SharePoint

**Comment by xugao** [comment]
@Jahnp - thanks for reporting! let me check with design on what the best experience should be. multi-line truncation can potentially cause perf problem so I'm not sure if we should do that within the control. just to point out, we have `onRenderLabel` prop which should allow you work around the problem while I explore the fix here.

**Comment by xugao** [comment]
@Jahnp - actually, it's little unclear to me what expected experience you are expecting here with truncated long text. pls take a look at [this PR](https://github.com/OfficeDev/office-ui-fabric-react/pull/11862) and let me know if it resolves the issue.

**Comment by Jahnp** [comment]
@xugao wow, awesome—thank you for the *very* swift response. Agreed that the ideal solution may be unclear and probably merits a spec, but I checked out your PR and also agree with the assumptions you made re: truncation (classic problem). And, thanks for the tip re: `onRenderLabel`—I should have looked more closely at the interface.

Truthfully, I filed this issue on behalf of a product team who I believed simply viewed the text clipping as the bug, which your PR fixes :) I don't think there was a specific expectation around the proper truncation behavior.
